addons/website/views/website_templates.xml

```diff
@@ -2589,6 +2589,61 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </xpath>
 </template>
 
+<template id="website.step_wizard" name="Step Wizard">
+    <!-- REQUIRED WHEN CALLING -->
+    <!-- o_wizard_activestep = int active step (from 0) -->
+    <!-- o_wizard_steps_text = array of step text -->
+    <!-- o_wizard_steps_href = array of step href -->
+    <!-- o_wizard_back_default = <- back href if @first step of the stepper -->
+
+    <!-- OPTIONAL -->
+    <!-- o_wizard_back_custom = <- back href if @any step of the stepper -->
+
+    <t t-set="o_wizard_steps_in_deg" t-value="(o_wizard_activestep + 1) / len(o_wizard_steps_text) * 360"/>
```

Comment:
You can simplify contextual variable names to improve readability.
eg `o_wizard_activestep` ->`activestep` (or `_activestep`)